macaroonbakery/utils.py

```diff
@@ -121,7 +122,7 @@ def cookie(
     port = str(u.port) if u.port is not None else None
     secure = u.scheme == 'https'
     if expires is not None:
-        expires = expires.strftime("%s")
+        expires = (expires - datetime(1970, 1, 1)).total_seconds()
```

Comment:
Please document that the expires parameter is expected to be a naive datetime in UTC.
Also it might be nice to explicitly check that expires has no timezone rather than rely on the check in the subtract method, which will produce a rather hard to diagnose exception.

Reply:
done